### PR TITLE
d.rast.edit: fix launch wxGUI (py3/wxPy4)

### PR DESCRIPTION
**Pull request by tmszi**
Fix launch d.rast.edit wxGUI (py3/wxPy4).

**Expected behavior:**

d_rast_edit_wxgui_exp

### CODE REVIEW

File: scripts/d.rast.edit/d.rast.edit.py [hunk 1]
```diff
@@ -79,6 +79,11 @@
 import atexit
 import grass.script as grass
 
+from grass.script.setup import set_gui_path
+set_gui_path()
+
+from gui_core.wrap import ClientDC, Menu, Panel
+
```
Comment (petrasovaa) [inline scripts/d.rast.edit/d.rast.edit.py:86]: Currently, the compilation fails, this probably needs to be in the try except below.

**Review by petrasovaa — CHANGES_REQUESTED** [review]

File: scripts/d.rast.edit/d.rast.edit.py (hunk 1 above)
Comment (tmszi) [reply scripts/d.rast.edit/d.rast.edit.py]: Fixed it in the [3819af9](https://github.com/OSGeo/grass/pull/1085/commits/3819af92e73871ca5fbd197d6ede0e4a1a117ca3).

**Review by petrasovaa — APPROVED** [review]
Thanks! It starts and looks ok. It doesn't seem to save the changes, but that looks like a different problem, maybe?

**Comment by tmszi** [comment]
> Thanks! It starts and looks ok. It doesn't seem to save the changes, but that looks like a different problem, maybe?

Saving a new raster map worked during my testing, only the controls are probably a little unclear.

Steps how to save the result raster map:

1. `d.rast.edit input=elevation output=edited_elevation`
2. Change New value (TextCtrl widget) value to e.g. 99999, and hit Enter keyboard
3. Mark cell(s) which you want change with left mouse click, 
4. Go to menu File -> Save (or Ctrl + S keyboard shortcut)
5. Check result raster map max. value `r.info -r map=edited_elevation`
6. Check number of cells you changed `r.mapcalc "extracted_elevation = if(edited_elevation==99999, 1, null())"`
7. `r.info -s map=extracted_elevation`

**Comment by petrasovaa** [comment]
I see, the problem is you need to press enter after editing the value, that is quite unintuitive in my opinion, I just expected that it's enough to change it. Also, when I use floating point number it reverts to asterisk. Did you understand what the color widget is supposed to do? Anyway, you can merge this PR and perhaps open a new one if we want improve the behavior.

**Comment by tmszi** [comment]
> I see, the problem is you need to press enter after editing the value, that is quite unintuitive in my opinion, I just expected that it's enough to change it. Also, when I use floating point number it reverts to asterisk. Did you understand what the color widget is supposed to do? Anyway, you can merge this PR and perhaps open a new one if we want improve the behavior.

I agree that confirming the value with the Enter key is unintuitive (I also had to read the man page first). I confirm that entering a value with a float point doesn't work. Color widget indicate null value same as asterisk in the TextCtrl widget (you get cell value with right click on the cell). Yes, I also agree that it needs to be improved.

**Comment by petrasovaa** [comment]
@tmszi I suggest to merge and backport

**Comment by tmszi** [comment]
> @tmszi I suggest to merge and backport

Ok, I merged it and backported.

**Pull request merged**